pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -374,6 +374,7 @@ func (lbaas *LbaasV2) EnsureLoadBalancer(clusterName string, apiService *api.Ser
 	loadbalancer, err := loadbalancers.Create(lbaas.network, createOpts).Extract()
 	if err != nil {
 		// cleanup what was created so far
+		glog.Warningf("EnsureLoadBalancer: loadbalancers.Create(%+v) error causing cleanup: %v", createOpts, err)
```

Comment:
The wording of the error is very code-oriented.  It's a personal style thing, but I feel something as visible as a warning (or error) should stand apart from the current implementation.

I'd suggest the simpler:
```go
glog.Errorf("Error creating loadbalancer %v: %v", createOpts, err)
```

Also note, this is an error (we're aborting the user's request because of this) so it should be logged as an error (not a warning) and use the word "Error" (as you have already done).